### PR TITLE
chore(konvoy): add airgapped upgrade

### PR DESCRIPTION
**Pull request by jkoelker**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-84226

## Description of changes being made

Add section to document additional steps needed to successfully upgrade an air-gapped cluster

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4551.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
@msdolbey @gflorinskaya will need thorough testing here.

**Comment by msdolbey** [comment]
@jkoelker - I just took a minute to change the yaml (I think what happened was everything was indented one extra space) 
If there are any changes, you'll want to pull again.

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+layout: layout.pug
+navigationTitle: Prepare Air-Gapped Cluster
+title: Prepare Air-Gapped Cluster
+menuWeight: 10
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md:5]: This should probably either be 5 or 15 
whether it should be before or after the "Prepare Local Environment" topic in the nav.

```suggestion
menuWeight: 15
```

Comment (jkoelker) [reply pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md:5]: Totes, that makes more sense to have it pinned after the local env instructions.

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Using the [Konvoy Image Builder](../../../image-builder), you can copy the requi
 1.  Download the PIP packages. This bundle includes a few packages required by DKP to bootstrap machines.
 
     ```bash
-    curl --output artifacts/pip-packages.tar.gz --location https://downloads.d2iq.com/konvoy/airgapped/pip-packages/pip-packages.tar.gz
+    curl --output artifacts/pip-packages.tar.gz --location https://downloads.d2iq.com/dkp/airgapped/pip-packages/pip-packages.tar.gz
```
Comment (jkoelker) [inline pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md:57]: While running through this while *only* using the 2.1 docs, I found that the pip packages don't exist in the `konvoy` prefix.

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        shim_debug = false
+      [plugins."io.containerd.runtime.v2.task"]
+        platforms = ["linux/amd64"]
+      [plugins."io.containerd.service.v1.diff-service"]
+        default = ["walking"]
+      [plugins."io.containerd.snapshotter.v1.devmapper"]
+        root_path = ""
+        pool_name = ""
+        base_image_size = ""
+        ```
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md:195]:
```suggestion
    ```
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+          service:
+            name: containerd
+            state: restarted
+            enabled: true
+    EOF
+    ```
+
+1.  Replay the current `/etc/containerd/config.toml` on the nodes.
+
+    Run this command:
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md:229]:
```suggestion
1.  Replay the current `/etc/containerd/config.toml` onto the nodes, by running this command:
```

**Review by cneth — APPROVED** [review]

**Pull request merged**